src/Component.js

```diff
@@ -41,11 +39,11 @@ export class CopyToClipboard extends React.PureComponent {
       text: _text,
       onCopy: _onCopy,
       options: _options,
+      onClickCapture,  // eslint-disable-line
       children,
-      ...props
+      ...restProps
     } = this.props;
-    const elem = React.Children.only(children);
 
-    return React.cloneElement(elem, {...props, onClick: this.onClick});
+    return React.cloneElement(children, {...restProps, onClick: this.onClick});
```

Comment:
Hmm, what is the use case for an array of children? Most of the time one can just wrap them in a container, e.g. `div` without much hassle.

Also ,n preact, `children` is always an array, yet this works as far as I can tell.